.eslintrc.js

```diff
@@ -25,6 +25,7 @@ module.exports = {
     'max-len': ['error', { code: 120 }],
     'no-dupe-class-members': 'off',
     'no-useless-constructor': 'off',
+    'no-undef': 'off',

```

Comment:
Why is this necessary?

Reply:
https://github.com/briza-insurance/wegood/issues/43